modules/apps/object/object-test/src/testFunctional/tests/ObjectPortlet.testcase

```diff
@@ -1014,12 +1014,52 @@ definition {
 	}
 
 	@description = "LPS-135396 - Verify it is possible to edit a Picklist entry on layout"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CanEditPicklistEntryOnLayout {
+		Picklist.addPicklistViaAPI(picklistName = "Custom Picklist");
+
+		for (var itemsName : list "1,2") {
+			Picklist.addPicklistItemViaAPI(
+				itemKey = "${itemsName}",
+				itemName = "Item Test ${itemsName}",
+				picklistName = "Custom Picklist");
+		}
+
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object",
+			objectName = "CustomObject",
+			pluralLabelName = "Custom Objects");
+
+		ObjectAdmin.openObjectAdmin();
+
+		ObjectPortlet.selectCustomObject(label = "Custom Object");
+
+		ObjectAdmin.goToFieldsTab();
+
+		ObjectAdmin.addObjectFieldViaUI(
+			fieldLabel = "Field Picklist",
+			fieldPicklist = "Custom Picklist",
+			fieldType = "Picklist");
```

Comment:
Hi @andreefarias 

You need to create a picklist field using API macro for that. Here is a macro for this: `ObjectAdmin.addObjectFieldViaAPI`. Let me know in case this macro doesn't work.